audio-worklet/basic/js/noise-generator.js

```diff
@@ -11,12 +13,17 @@ class NoiseGenerator extends AudioWorkletProcessor {
     super(options);
   }
 
-  process(input, output, parameters) {
-    let outputChannelData = output.getChannelData(0);
+  process(inputs, outputs, parameters) {
+    let output = outputs[0];
     let amplitude = parameters.amplitude;
-    for (let i = 0; i < 128; ++i) {
-      outputChannelData[i] = Math.random() * amplitude[i];
+    for (let channel = 0; channel < output.length; ++channel) {
+      let outputChannel = output[channel];
+      for (let i = 0; i < outputChannel.length; ++i) {
+        outputChannel[i] = Math.random() * amplitude[i];
```

Comment:
Probably already mentioned this earlier, but Math.random returns a value in the range [0, 1).  I think a noise generator should have zero mean.

Reply:
Oops. Fixed.